client/src/components/CourseInfo/CourseInfo.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { classnames } from "../../Utilities";
+
+interface CourseInfoProps {
+  hours: string;
+  department: string;
+  description: string;
+}
+
+const BentoBase = [
+  "flex",
+  "flex-col",
+  "p-4",
+  "my-2",
+  "rounded-3xl",
+  "bg-primary-95",
```

Comment:
`bg-secondary-95`

Reply:
(In case you're wondering, it is intentionally designed to have color variantion with bento)

Reply:
Looks good